test/unit/config/article_scrape_patterns_test.rb

```diff
@@ -56,8 +56,28 @@ def test_scrape_methods_presets_are_valid
     end
   end
 
+  def test_scrape_methods_have_proper_variable_names
```

Comment:
I was on the fence with it. It adds a lot of value, but is a bit meta.
I tried to keep it to just checking the method names